locking/locks.go

```diff
@@ -154,6 +157,17 @@ func (c *Client) UnlockFileById(id string, force bool) error {
 	if err := c.cache.RemoveById(id); err != nil {
 		return fmt.Errorf("Error caching unlock information: %v", err)
 	}
+	path := unlockRes.Lock.Path
+	gitRoot, err := git.RootDir()
+	if err != nil {
+		return err
+	}
+	absPath := filepath.Join(gitRoot, path)
+
+	// Make non-writeable if required
+	if c.SetLockableFilesReadOnly && c.IsFileLockable(path) {
```

Comment:
Not sure if this comment refers to initializing the Config.SetLockableFilesReadOnly to true by default? If so, I believe the docs already say that's how it works. I assumed the config variable (and env var) were added as part of a future method for configuring this behaviour. I noticed other places that do not test the flag and were already setting the readonly flag (despite the config defaulting to false), so there's more fixes to be done.

I'll look at adding the test.